Implementation/core/src/main/java/com/teamonehundred/pixelboat/BoatRace.java

```diff
@@ -35,6 +39,7 @@
     protected Texture bleachers_r;
 
     protected List<CollisionObject> obstacles;
+    protected List<CollisionObject> powerups;
```

Comment:
Could powerups be part of the obstacles list? If there is a particular reason they are not it's fine.
Putting them in the obstacles list would make it easier for the GameState object to save everything

Reply:
resolved, no changes to be made